src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -2807,7 +2818,7 @@ def assign_express_route_port_identity(cmd, resource_group_name, express_route_p
     ports = client.get(resource_group_name, express_route_port_name)
 
     ManagedServiceIdentity, ManagedServiceIdentityUserAssignedIdentitiesValue = \
-        cmd.get_models('ManagedServiceIdentity', 'ManagedServiceIdentityUserAssignedIdentitiesValue')
+        cmd.get_models('ManagedServiceIdentity', 'Components1Jq1T4ISchemasManagedserviceidentityPropertiesUserassignedidentitiesAdditionalproperties')  # pylint: disable=line-too-long
```

Comment:
>Components1Jq1T4ISchemasManagedserviceidentityPropertiesUserassignedidentitiesAdditionalproperties [](start = 50, length = 98)

It's an autorest/swagger issue, and will fix it future PR once the SDK is fixed.